addons/web_editor/static/src/js/backend/convert_inline.js

```diff
@@ -346,16 +346,24 @@ function classToStyle($editable, cssRules) {
             delete css['font-family'];
         }
 
-        // Do not apply css that would override inline styles (which are prioritary).
-        let style = node.getAttribute('style') || '';
-        // Outlook doesn't support inline !important
-        style = style.replace(/!important/g,'');
+        const stylesObject = {};
```

Comment:
This refactoring has a side-effect that breaks tests (order of styles is different). Maybe not necessary to do the ref here, it's incomplete anyway but good idea for a later PR.